source/gui/configProfiles.py

```diff
@@ -337,6 +338,17 @@ def saveTriggers(self, parentWindow=None):
 				parent=parentWindow,
 			)
 
+	def GetFontFromConfig(self) -> wx.Font:
```

Comment:
This shouldn't be a duplicated function across so many files. Please just add a single `getFontFromConfig` to `gui.nvdaControls`